guide/intro/chapter-1/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: What is Pygame
+next_page: [/guide/intro/chapter-2, Opening A Window]
+---
+
+<body>
+  <p>In this article, we'll go over what pygame is and how to install it.</p>
+
+  <h2>Table of contents</h2>
+  <p></p>
+  <ol id="toc">
+    <!--
+            To be filled by javascript
+            -->
+  </ol>
+
+  <h2><a id="introduction" class="js">Introduction</a></h2>
+  <p>
+    It's time to get started on your pygame journey, but what is pygame? Well
+    pygame is "a free and open-source cross-platform library for the development
+    of multimedia applications like video games using Python". As the
+    description suggests, it's mostly used to make video games - which is why in
+    this book we'll be learning techniques relevant to making games
+  </p>
+
+  <p>
+    Pygame provides the very basic utility to draw and manipulate pixels on to
+    the screen, much like <a href="https://www.libsdl.org/">SDL2</a>, but with
+    extended features in various forms that makes it easier to use. Given this
+    nature, you generally have a lot of control on how you want your code to
+    run, so it may have a bit of a learning curve. But fear not! After reading
+    this book you should be able to use pygame just fine, as long as you get
+    your practice in 🔫
+  </p>
+
+  <p>
+    In this book we'll aim to cover how to do basic things with pygame, with
+    little exercises in between to make sure you're not falling off, goodluck
+  </p>
+
+  <h2><a id="pygame-ce" class="js">Pygame-CE</a></h2>
+  <p>
+    So pygame currently has two major distributors, the original one and a fork
```

Comment:
Starting a sentence with "So" feels wrong, suggest opening with something different.

```suggestion
    In the event that you weren't aware, pygame currently has two major distributions, the original one and a fork
```

Reply:
> In the event that you weren't aware 

I feel like this is too formal? In English it's valid to start sentences with "So" afaik. In formal writing it's frowned upon - but I was trying to go for a more friendly and engaging vibe for the whole article/book. This isn't necessarily unfriendly, of course, but I feel being too formal makes it less engaging
But if we really dont want to move forward with "So", I think we should look for an alternative as "In the event that you weren't aware" feels too formal 
I'd say "In case you didn't know, " or  just starting off with "Pygame has two major distributions" (but I do feel like adding "So" would be more engaging, we should poll on this maybe)